pkg/manager/manager.go

```diff
@@ -121,9 +125,31 @@ func Login(ctx context.Context, password, address string) (user string, pass str
 			return "", "", ctx.Err()
 		}
 	}
+
+	store, err := credentials.NewStore(cfg, nil)
+	if err != nil {
+		return user, pass, err
+	}
+
+	if err = store.Add(ctx, apiv1.Credential{
+		ServerAddress: address,
+		Username:      user,
+		Password:      &pass,
+		LocalStorage:  true,
+	}, true); err != nil {
+		return user, pass, err
+	}
```

Comment:
What does this new store refer to? Is it just storing the credentials in the local CLI config file?

Reply:
Yes, that's correct.

Before this change, the login `Run` function would call `Login` and then store the credential. I moved this is in here so that I could use the `Login` function in the new place and get the new credential stored automatically.

I hope that makes sense.